server/mdm/android/service/androidmgmt/google_client.go

```diff
@@ -162,8 +165,32 @@ func (g *GoogleClient) createPubSub(ctx context.Context, pushURL string) (string
 	return topic.String(), nil
 }
 
+func policyFieldMask() string {
+	getJSONFieldName := func(t string) string {
+		t = strings.TrimSuffix(t, ",omitempty")
```

Comment:
I think you can safely use `strings.Cut` https://pkg.go.dev/strings#Cut with the comma as separator, and the first return value will be the JSON field name (or empty).